pkgs/development/python-modules/requests/default.nix

```diff
@@ -33,6 +33,9 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  # AttributeError: 'KeywordMapping' object has no attribute 'get'
+  doCheck = ! isPy27;
```

Comment:
That's not a solution either.
The failing test probably points to some dependency not being compatible with Python 2.

Reply:
That error is raise in `pytest`, so it's not an issue for `requests` itself.

Reply:
I know, but means that one of the dependencies (maybe pytest) is not compatible with Python 2.

Reply:
If upstream dropped python2 support we should add the latest compatible version for python 2 to nixpkgs as long as it is needed to not have any surprises or strange breakages.

Reply:
Requests still supports it and so does the version of pytest we package for Python 2.